.github/workflows/build-sample-apps.yml

```diff
@@ -104,11 +104,14 @@ jobs:
         lane: 'android build'
         subdirectory: "samples/${{ matrix.sample-app }}"
       env:
-        ANDROID_SIGNING_ALIAS: ${{ secrets.ANDROID_SIGNING_ALIAS }}
-        ANDROID_SIGNING_KEY_PASSWORD: ${{ secrets.ANDROID_SIGNING_KEY_PASSWORD }}
-        ANDROID_SIGNING_STORE_PASSWORD: ${{ secrets.ANDROID_SIGNING_STORE_PASSWORD }}
         FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64: ${{ secrets.FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64 }}
     
+    - name: Save built app binary for manual install if needed. 
```

Comment:
I read the Slack conversation, I am not sure I understand the use case for this. if the APK is available at Firebase and here what's the difference? 

previously the problem was APK signing, while installing, that problem would still be applicable here and Firebase. 

maybe I am missing something?

Reply:
+1 to @Shahroz16. As long as the same builds are being uploaded on Firebase, I don't think we need them at Github as they are eventually same and we can download them directly from Firebase if needed.

Reply:
> if the APK is available at Firebase and here what's the difference?

Ah, didn't think of that. I forgot that you can download APK files from firebase.